app/src/modules/content/routes/item.vue

```diff
@@ -150,7 +150,6 @@
 
 		<v-form
 			ref="form"
-			:key="collection"
```

Comment:
IIRC this was added as a fix via #8781, but it does seems like I'm not able to reproduce #8753 even after removing this key. But wanted to bring this up just in case anyone else can help verify this on their end 👍

Reply:
I can't reproduce this either. Looks like Vue recreates the `input-rich-text-html` component each time you switch the component.

Reply:
Maybe something that changed with a newer version of Vue / TinyMCE 🤔 Do you agree that we can leave it as is for now and take a new look at it if it turns out that it's still a problem?

Reply:
> Do you agree that we can leave it as is for now and take a new look at it if it turns out that it's still a problem?

Yeah sounds good to me! 👍